lib/adapters/class-adapter.php

```diff
@@ -181,7 +181,13 @@ public function add_phrase_matching_to_es_args( array $es_args ): array {
 			return $es_args;
 		}
 
+		// TODO This isn't working as expected when using WP_Query to trigger a search from the context of the test suite.
 		$search = get_query_var( 's' );
+		// If query vars aren't set, then try to pull this from the actual ES Query instead of the WP Query Object.
+		// This may only work for VIP Search.
+		if ( empty( $search ) ) {
+			$search = $es_args['query']['function_score']['query']['bool']['should'][0]['multi_match']['query'] ?? '';
+		}
```

Comment:
@kevinfodness / @mboynes I'd love your feedback on this. I cam across this when running tests, which use a new WP_Query instance that passes a value to `'s'` to simulate a search. While L185 above works on sites where we are making searches, it isn't functioning during tests because the `global $wp_query` object is not setup. 

This got me thinking that there may be instances where we would want to ensure that phrase matching extended to ES-integrated WP_Query instances, and so we should cover it in the Adapter. This left a bad taste in my mouth, though. 

Should this code pull from the query vars or look directly in the `$es_args` that are passed to it for the actual query string?

Reply:
I'd recommend reconfiguring this repo to use Mantle Testkit which has better support for simulating certain types of actions, like visiting the search page, and will set up globals for you.

Reply:
good news! This is already using Mantle Test Kit, since the pest packages use the test framework :) Thank you for the suggestion on using the `get()` functionality to set this up. I will dig into this now.

Reply:
using `$this->get( '/?s="exact match"' );` in the tests works like a charm, thanks!